### ISSUE TITLE
error when filter in tickets : task group in charge : glpi 9.5.1

### ISSUE THREAD

**Issue by mdesfawes** [glpi-project/glpi#7842]
Dear All,

in tickets, when i will put a filter on "tasks : group in charge", i receive this error : 

Uncaught Exception TypeError: Argument 1 passed to Dropdown::addNewCondition() must be of the type array, string given, called in /var/www/html/glpi/ict/inc/dropdown.class.php on line 157 in /var/www/html/glpi/ict/inc/dropdown.class.php at line 278

Thanks for your help
Michel

**Comment by trasher** [comment]
Informations requested in ticket template has not been provided.

**Comment by mdesfawes** [comment]
**Describe the bug**
in tickets, when i will put a filter on "tasks : group in charge", i receive this error :

Uncaught Exception TypeError: Argument 1 passed to Dropdown::addNewCondition() must be of the type array, string given, called in /var/www/html/glpi/ict/inc/dropdown.class.php on line 157 in /var/www/html/glpi/ict/inc/dropdown.class.php at line 278
**Page(s) URL**
http://glpi-dev/glpi/front/ticket.php
**To reproduce**
add a filter with task : group in charge
**Expected behavior**
i cannot complete the filter, error occurs before
**Logs**

```
paste logs here
Find them in *-error.log files under glpi/files/_log/
```

[2020-08-12 11:17:24] glpiphplog.CRITICAL:   *** Uncaught Exception TypeError: Argument 1 passed to Dropdown::addNewCondition() must be of the type array, string given, called in /var/www/html/glpi/ict/inc/dropdown.class.php on line 157 in /var/www/html/glpi/ict/inc/dropdown.class.php at line 278
  Backtrace :
  inc/dropdown.class.php:157                         Dropdown::addNewCondition()
  inc/commondbtm.class.php:3916                      Dropdown::show()
  inc/commondbtm.class.php:4837                      CommonDBTM::dropdown()
  inc/ticket.class.php:6789                          CommonDBTM->getValueToSelect()
  inc/search.class.php:3017                          Ticket->getValueToSelect()
  inc/search.class.php:2911                          Search::displaySearchoptionValue()
  ajax/search.php:58                                 Search::displaySearchoption()
  
[2020-08-12 11:20:13] glpiphplog.CRITICAL:   *** Uncaught Exception TypeError: Argument 1 passed to Dropdown::addNewCondition() must be of the type array, string given, called in /var/www/html/glpi/ict/inc/dropdown.class.php on line 157 in /var/www/html/glpi/ict/inc/dropdown.class.php at line 278
  Backtrace :
  inc/dropdown.class.php:157                         Dropdown::addNewCondition()
  inc/commondbtm.class.php:3916                      Dropdown::show()
  inc/commondbtm.class.php:4837                      CommonDBTM::dropdown()
  inc/ticket.class.php:6789                          CommonDBTM->getValueToSelect()
  inc/search.class.php:3017                          Ticket->getValueToSelect()
  inc/search.class.php:2911                          Search::displaySearchoptionValue()
  ajax/search.php:58                                 Search::displaySearchoption()
  
[2020-08-12 11:20:33] glpiphplog.CRITICAL:   *** Uncaught Exception TypeError: Argument 1 passed to Dropdown::addNewCondition() must be of the type array, string given, called in /var/www/html/glpi/ict/inc/dropdown.class.php on line 157 in /var/www/html/glpi/ict/inc/dropdown.class.php at line 278
  Backtrace :
  inc/dropdown.class.php:157                         Dropdown::addNewCondition()
  inc/commondbtm.class.php:3916                      Dropdown::show()
  inc/commondbtm.class.php:4837                      CommonDBTM::dropdown()
  inc/ticket.class.php:6789                          CommonDBTM->getValueToSelect()
  inc/search.class.php:3017                          Ticket->getValueToSelect()
  inc/search.class.php:2911                          Search::displaySearchoptionValue()
  ajax/search.php:58                                 Search::displaySearchoption()
  

**Screenshots**

If applicable, add screenshots to help explain your problem.


**Your GLPI setup (you can find it in Setup > General menu, System tab)**

```
paste here
```

**Additional context**

Add any other context about the problem here.

**Comment by okvok** [comment]
![image](https://user-images.githubusercontent.com/20252855/90313647-e86db580-df16-11ea-9c59-2fd502a2810c.png)

**Comment by cedric-anne** [comment]
Hi,

Issue was fixed by #7768.

Regards
